content/collections.md

```diff
@@ -358,12 +358,17 @@ Migrations.add({
       batch.find({_id: list._id}).updateOne({$set: {todoCount}});
     });
 
-    // We need to wrap the async function to get a synchronous API that migrations expects
-    const execute = Meteor.wrapAsync(batch.execute, batch);
-    return execute();
+    //Mongo throws an error if we execute a batch operation without actual operations, e.g. when Lists was empty.
+    if(batch.s && batch.s.currentBatch && batch.s.currentBatch.operations && batch.s.currentBatch.operations.length > 0){
```

Comment:
Wouldn't it be more appropriate (i.e. safer) to use the (documented) [`Bulk.tojson` method](https://docs.mongodb.com/manual/reference/method/Bulk.tojson/#Bulk.tojson) to access this information instead of using the (to my knowledge, undocumented) `s.currentBatch` attributes?

Maybe the information presented isn't quite the same and I don't have a test script immediately available (maybe you do?), but it seems that `batch.tojson()` should return something along the lines of:

```
{ “nInsertOps” : 2, “nUpdateOps” : 0, “nRemoveOps” : 1, “nBatches” : 2 }
```

...where it might even be acceptable to just check if `nBatches > 0` before preceding.